http.c

```diff
@@ -299,9 +301,11 @@ evhttp_htmlescape(const char *html)
  * equivalent HTTP command, or NULL if the evhttp_command_type is
  * unrecognized. */
 static const char *
-evhttp_method(enum evhttp_cmd_type type)
+evhttp_method(struct evhttp_connection *evcon,
```

Comment:
Maybe return `evhttp_extended_methods` and reuse it in `evhttp_method_may_have_body` for example?